examples/asr/hkust/mtl_transformer.json

```diff
@@ -34,19 +34,20 @@
 
   "decode_config":{
     "beam_search":true,
-    "beam_size":4,
-    "ctc_weight":0.3,
-    "lm_weight":0.1,
-    "lm_path":"examples/asr/hkust/data/4gram.arpa"
+    "beam_size":10,
+    "ctc_weight":0.5,
+    "lm_type":"ngram",
+    "lm_weight":0.3,
+    "lm_path":"examples/asr/hkust/data/lm.bin"
```

Comment:
Since you changed the language model to binary format, you should update run.sh to create lm.bin instead of 4gram.arpa

Reply:
sure